app/views/landing_pages/TOA_en.html.erb

```diff
@@ -0,0 +1,30 @@
+<h1>Policy and Disclosure Statement</h1>
```

Comment:
@verynear I'm not sure why the privacy policy documents are part of this Pull Request?  I made some changes to these in a previous PR, so it's possible you got out of sync?  Do you think you could remove these changes?